### PR TITLE
[index.md] rebrand Wootric to InMoment

### PR DESCRIPTION
**Pull request by tcgilbert**
### Proposed changes

The documentation is largely similar, only changed mentions of "Wootric" to "InMoment"

### Merge timing

asap

### Related issues (optional)

Pull Request for changing the Wootric destination docs: https://github.com/segmentio/segment-docs/commit/fa6154264ed8d207263e5fbb9fe04dbfc33f7c1a

### CODE REVIEW

File: src/connections/sources/catalog/cloud-apps/wootric/index.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
 ---
-title: Wootric
+title: InMoment (formerly Wootric)
 rewrite: true
 ---
 
-[Wootric](https://wootric.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is modern customer feedback management software. Easily incorporate Voice of Customer data in your decision-making processes and marketing campaigns, and trigger customer follow-up when you send survey responses to your CRM, marketing platform or data warehouses automatically.
+[InMoment (formerly Wootric)](https://wootric.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is modern customer feedback management software. Easily incorporate Voice of Customer data in your decision-making processes and marketing campaigns, and trigger customer follow-up when you send survey responses to your CRM, marketing platform or data warehouses automatically.
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/wootric/index.md:6]:
```suggestion
[InMoment](https://wootric.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) (formerly Wootric) is modern customer feedback management software. You can incorporate Voice of Customer data in your decision-making processes and marketing campaigns, and trigger customer follow-up when you send survey responses to your CRM, marketing platform or data warehouses automatically.
```

File: src/connections/sources/catalog/cloud-apps/wootric/index.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 
-This source is maintained by Wootric. For any issues with the source, [contact the Wootric support team](mailto:support@wootric.com).
+This source is maintained by InMoment. For any issues with the source, [contact the InMoment support team](mailto:support@wootric.com).
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/wootric/index.md:8]:
```suggestion
InMoment maintains this source. For any issues, [contact the InMoment support team](mailto:support@wootric.com).
```

File: src/connections/sources/catalog/cloud-apps/wootric/index.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 
 ## Getting Started
 
 1. From your Segment UI’s Sources page click on “Add Source”.
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/wootric/index.md:12]:
```suggestion
1. In Segment go to **Connections** and click **Add Source**.
```

File: src/connections/sources/catalog/cloud-apps/wootric/index.md [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
-2. Search for "Wootric" within the Sources Catalog and confirm by clicking "Connect".
+2. Search for "InMoment" within the Sources Catalog and confirm by clicking "Connect".
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/wootric/index.md:13]:
```suggestion
2. Search for **InMoment** in the Source Catalog and confirm by clicking **Connect**.
```

File: src/connections/sources/catalog/cloud-apps/wootric/index.md [hunk 5]
```diff
@@ -1,22 +1,22 @@
...
 3. Give the Source a nickname and follow the set up flow to "Add Source". The nickname will be used to designate the source in the Segment interface, and Segment will create a related schema name. The schema name is the namespace you'll be querying against in your warehouse. The nickname can be whatever you like, but we recommend sticking to something that reflects the source itself and distinguishes amongst your environments (eg. SourceName_Prod, SourceName_Staging, SourceName_Dev).
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/wootric/index.md:14]:
```suggestion
3. Give the Source a nickname and click **Add Source**. The nickname is used to designate the source in the Segment interface, and Segment creates a related schema name. The schema name is the namespace you query against in your warehouse. The nickname can be whatever you like, but Segment recommends using a name that reflects the source itself and distinguishes amongst your environments (eg. SourceName_Prod, SourceName_Staging, SourceName_Dev).
```

File: src/connections/sources/catalog/cloud-apps/wootric/index.md [hunk 6]
```diff
@@ -1,22 +1,22 @@
...
-4. Copy the Write key from the Segment UI and log in to your Wootric account - navigate to Settings > Integrations > Send Wootric Data > Send survey responses back to Segment and paste the key to connect.
+4. Copy the Write key from the Segment UI and log in to your InMoment account - navigate to Settings > Integrations > Send InMoment Data > Send survey responses back to Segment and paste the key to connect.
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/wootric/index.md:15]:
```suggestion
4. Go to the **Settings > API Keys**. 
5. Copy the **Write Key** and log in to your InMoment account. 
6. Navigate to **Settings > Integrations > Send InMoment Data > Send survey responses back to Segment** and paste the write key to connect.
```

File: src/connections/sources/catalog/cloud-apps/wootric/index.md [hunk 7]
```diff
@@ -47,4 +47,4 @@ Now that your Source is set up, you can connect it with Destinations.
 
 Log into your downstream tools and check to see that your events are populating and they contains all the properties you expect. If all your events and properties are not showing up, refer to the Destination docs for troubleshooting.
```
Comment (stayseesong) [inline src/connections/sources/catalog/cloud-apps/wootric/index.md:48]:
```suggestion
Log into your downstream tools and check to see that your events are populating and they contain all the properties you expect. If all your events and properties are not showing up, refer to the Destination docs for troubleshooting.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Comment by stayseesong** [comment]
@tcgilbert I noticed that "InMoment" isn't currently a search option in the Source catalog but Wootric still is

**Comment by markzegarelli** [comment]
@tcgilbert Can you change the name in Partner Portal for this source so that we can merge?

**Review by stayseesong — APPROVED** [review]

**Pull request merged**